doc/Language/classtut.pod6

```diff
@@ -276,10 +264,9 @@ The third attribute represents the state of completion of a task:
 =for code
 has Bool $.done;
 
-X<|twigils,.>
-X<|twigils,accessors>
-X<|accessor methods>
-X<|classes,accessors>
+X<|Tutorial,.>
+X<|Tutorial,accessors>
```

Comment:
Hum. I really don't know here. It's not about accessors, it's about accessors that use twigils. You're losing that information here.

Reply:
No, there is no "twigil accessors" category.

Categorization is complex, I'm sure you're aware of that better than anyone. You cannot be too "high-level" and at the same time you cannot categorize every tiniest detail creating hundreds of categories to be lost.

This page is a tutorial for everything related to OOP and in this paragraph accessors are explained, so "accessors in category Tutorial" looked fitting to me.

Reply:
Again, this is not explaining accessors. It's explaining a specific type of accessors that use twigils. By dropping twigils you're negating the possibility of searching that, as opposed to simple accessors.

Reply:
>Again, this is not explaining accessors. It's explaining a specific type of accessors that use twigils

Ah, I see what you mean! Now I believe we have different views on the concept/purpose of index categorization.

Please, bear with me for a bit then.

In my view, we have an index of things to search. Things go from particular functions like `run` or `dir` to language-wide concepts like classes or twigils.

The idea is to categorize those things into certain buckets, so you can, say, search for all the methods starting with `is-` or all the `$` variables (but you don't need explanation of what $ twigil is, etc).

So buckets/categories are meant to be some sort of an umbrella to a meaningful number of entities. Say "subroutines" is a bucket for subroutines or "tutorial" is a bucket for places in the documentation when some term is explained in a tutorial-like manner for a quick dive.

You claim we need a "twigil accessors" bucket/category, but what belongs to this category? Only this anchor where the doc talks about twigil accessors? Even if there are other places like that (we can arguably sweep things like `$.` syntax there), there are but a few of them. Trying to store additional information on the labels of the buckets (we cannot have a bucket "accessors", but only "twigil accessors" and maybe some other types of accessors) creates not a lot less categories than we have items.

I agree with you the information the category gives is great (speaking of the other example, we have `is rw` in Tutorial section, `is rw` trait explanation for classes and in signatures, so you can decide if you want a tutorial or not), but maybe we don't agree on the minimal number of items necessary to start a new "bucket". IMO anything with less than say 15-20 items doesn't need a bucket by itself, but maybe the line is different for you?

So, in my view, categories are used to narrow the search query down as their primary purpose, and the additional information is the secondary purpose. The anchor itself should give the information when it's necessary.

Does it match your view on the issue in any way?

Reply:
No, I'm not saying that should be a category. It's simply an indexing term.